src/mlpack/core/util/arma_traits.hpp

```diff
@@ -160,7 +160,7 @@ struct GetRowType<arma::SpMat<eT>>
 template<typename MatType>
 struct GetColType
 {
-  typedef arma::Row<typename MatType::elem_type> type;
```

Comment:
Nice catch, thank you!

Reply:
You are welcome, it is my fault, I reviewed this and it was good for me :face_with_spiral_eyes: